### PR TITLE
Mtbench101 2oc

### PR DESCRIPTION
**Pull request by sefira**

### CODE REVIEW

File: configs/datasets/subjective/multiround/mtbench101_judge.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+from opencompass.openicl.icl_prompt_template import PromptTemplate
+from opencompass.openicl.icl_retriever import ZeroRetriever
+from opencompass.openicl.icl_inferencer import ChatInferencer, GenInferencer
+from opencompass.openicl.icl_evaluator import LMEvaluator
+from opencompass.datasets import MTBench101Dataset
+
+
+subjective_reader_cfg = dict(
+    # input_columns=['dialogue', 'capability', 'system_prompt', 'prompt_template'],
```
Comment (conceptmath) [inline configs/datasets/subjective/multiround/mtbench101_judge.py:9]: delete comments

File: configs/eval_subjective_mtbench101.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+api_meta_template = dict(
+    round=[
+        dict(role='SYSTEM', api_role='SYSTEM'),
+        dict(role='HUMAN', api_role='HUMAN'),
+        dict(role='BOT', api_role='BOT', generate=True),
+    ]
+)
+
+# api_meta_template = dict(
```
Comment (yifan123) [inline configs/eval_subjective_mtbench101.py:31]: delete comments

File: configs/eval_subjective_mtbench101.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        max_num_workers=32,
+        task=dict(type=OpenICLInferTask),
+    ),
+)
+
+# -------------Evalation Stage ----------------------------------------
+
+## ------------- JudgeLLM Configuration
+judge_models = [dict(
+    abbr='GPT4-Turbo',
```
Comment (yifan123) [inline configs/eval_subjective_mtbench101.py:86]: need to check the openai api type

File: configs/datasets/subjective/multiround/mtbench101_judge.py (hunk 1 above)
Comment (sefira) [reply configs/datasets/subjective/multiround/mtbench101_judge.py:9]: > delete comments

done

File: configs/eval_subjective_mtbench101.py (hunk 2 above)
Comment (sefira) [reply configs/eval_subjective_mtbench101.py:31]: > delete comments

done

**Pull request closed without merging**